Packs/RiskIQDigitalFootprint/IndicatorFields/incidentfield-riskiqassetenterpriseasset.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "indicator_riskiqassetenterpriseasset",
+    "version": -1,
+    "modified": "2020-07-08T19:53:45.519418543+05:30",
+    "name": "RiskIQAsset Enterprise Asset",
+    "ownerOnly": false,
+    "placeholder": "Has the asset been designated as an enterprise asset by RiskIQ Digital Footprint.",
+    "description": "Has the asset been designated as an enterprise asset by RiskIQ Digital Footprint.",
+    "cliName": "riskiqassetenterpriseasset",
+    "type": "shortText",
```

Comment:
@avidan-H We will be making this a singleSelect field with "Select" as its default and "true" and "false" as its potential values, since we do not want to keep any default value for this field. Please confirm if this approach is appropriate.